custom-plugins/grafana-worldmap-panel/src/worldmap.ts

```diff
@@ -138,6 +140,14 @@ export default class WorldMap {
   }
 
   updateCircles(data) {
+    console.log("Updating circles...");
```

Comment:
can you remove / comment the `console.log` please

Reply:
ok